masterlist.yaml

```diff
@@ -314,6 +314,10 @@ common:
     name: 'skse/plugins/po3_SpellPerkItemDistributor.dll'
     display: '[Spell Perk Item Distributor](https://www.nexusmods.com/skyrimspecialedition/mods/36869/)'
 
+  - &OAR
+    name: 'skse/plugins/OpenAnimationReplacer.dll'
+    display: '[Open Animation Replacer](https://www.nexusmods.com/skyrimspecialedition/mods/92109)'
```

Comment:
As with the `&DAR` PR, please sort `&OAR` alphabetically inbetween the other File Anchors, and add a trailing slash `/` at the end of the link.